### DISCUSSION TITLE
Help with Oqtane.docs

### DISCUSSION THREAD

**Discussion by markagama** [oqtane/oqtane.framework#1742] [Q&A]
Hi, I find that every time I come here, I'm asking questions that are likely to be FAQ.

So who do I speak with to help with the Oqtane.docs project?

I'm sure that everyone could do with a FAQ.

Mark

**Comment by sbwalker** [comment]
I would suggest that you submit a PR to the https://github.com/oqtane/oqtane.docs repository - which generates the docs at https://docs.oqtane.org/

**Reply by markagama** [reply]
@sbwalker I'm a bit out of touch with some of the shorthand, by PR do you mean pull request? I'm not sure what this will achieve? What I would like to do is to provide updates to this documentation.

**Reply by sbwalker** [reply]
@markagama the standard mechanism for contributing to open source projects on Github is to fork/clone the code, make your changes locally, and then submit a pull request which can be merged by the maintainer of the project. This process is used for both code and documentation and it offers the benefit of maintaining a record of all changes to project artifacts over time as well as a transparent peer review/feedback/approval workflow.
